tests/conftest.py

```diff
@@ -1,18 +1,45 @@
+# -*- coding: utf-8 -*-
+
 import pytest
-from funnel import app as hasgeekapp, db
-from funnel.models import Profile, Project, User
+import uuid
+from funnel import app
+from funnel.models import db, Profile, Project, User, Label, Proposal, Team
+
+
+@app.route('/usertest')
+def user_test():
+    from coaster.auth import current_auth
+    return current_auth.user.username if current_auth.user is not None else "anon"
```

Comment:
`"<anon>"` so that it has characters that are invalid in usernames. You could have a user named `anon`.